masonite/auth/Sign.py

```diff
@@ -38,11 +38,16 @@ def sign(self, value):
         
         Returns:
             string -- Returns the encrypted value.
-        """
 
-        f = Fernet(self.key)
-        self.encryption = f.encrypt(bytes(value, 'utf-8'))
-        return self.encryption.decode('utf-8')
+        Raises:
+            InvalidSecretKey -- Thrown if the secret key does not have correct padding.
+        """
+        try:
+            f = Fernet(self.key)
```

Comment:
This is prob more of a pet peeve of mine but can we just wrap the code that was throwing the exception? Can we do something like:

```python
try:
    f = Fernet(self.key)
except binascii.Error:
    raise InvalidSecretKey("The encryption key passed does not have correct padding and therefore cannot be used as a secret key.")

self.encryption = f.encrypt(bytes(value, 'utf-8'))
return self.encryption.decode('utf-8')
```